packages-exp/auth-exp/package.json

```diff
@@ -29,6 +29,7 @@
     "@firebase/app-types-exp": "0.x"
   },
   "dependencies": {
+    "@firebase/app": "^0.6.1",
```

Comment:
Versions should be fixed. Also, we should depend on `@firebase/app-exp` instead.

Reply:
Actually no need to list `@firebase/app-exp` in dependencies. It's already a peer dependency.